src/backend/tests/it/utils/mock.rs

```diff
@@ -13,10 +13,11 @@ pub const WEENUS_SYMBOL: &str = "Weenus";
+pub const VC_HOLDER: &str = "7eboi-tyuys-aqm4c-w2l7i-vgucm-xvawx-lemzx-6kq2g-f53u7-yvfh2-nae";
 pub const ISSUER_CANISTER_ID: &str = "qdiif-2iaaa-aaaap-ahjaq-cai";
 pub const ISSUER_ORIGIN: &str = "https://dummy-issuer.vc/";
-pub const II_CANISTER_ID: &str = "rdmx6-jaaaa-aaaaa-aaadq-cai";
+pub const VC_DERIVATION_ORIGIN: &str = "https://l7rua-raaaa-aaaap-ahh6a-cai.ic0.app";
+pub const II_CANISTER_ID: &str = "fgte5-ciaaa-aaaad-aaatq-cai";
```

Comment:
out of curiosity, why the II canister changed too for the mocks?

Reply:
Because I created the JWT with a different II.

Reply:
Ah understood! tks for the explanation!

Reply:
Are you going to approve it? Or should I ask for @peterpeterparker 's review specifically?

Reply:
honestly, to me it looks good, it seems trivial, but for this I would prefer to rely on @peterpeterparker or @bitdivine to review